.devcontainer/windows-host-helper.sh

```
@@ -39,13 +39,26 @@ tmpdir=$(mktemp -d)
-# Enable dotglob so that '*' includes hidden files
-shopt -s dotglob
 
-# Move all contents (including hidden files) from the cloned repo to the target folder
-mv "$tmpdir/$repoName"/* "./$repoName/"
+SRC="$(realpath "$tmpdir/$repoName")"
+DST="$(realpath "./$repoName")"
 
-# Clean up: remove the temporary directory
+# 1) Recreate directory tree under $DST
+while IFS= read -r -d '' dir; do
+  rel="${dir#$SRC/}"             # strip leading $SRC/ → e.g. "sub/dir"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)